repo/db/sales.go

```diff
@@ -273,3 +278,41 @@ func (s *SalesDB) Count() int {
 	row.Scan(&count)
 	return count
 }
+
+func (s *SalesDB) GetNeedsResync() ([]repo.UnfundedSale, error) {
+	s.lock.RLock()
+	defer s.lock.RUnlock()
+	var ret []repo.UnfundedSale
+	rows, err := s.db.Query(`select orderID, timestamp from sales where state=? and needsSync=?`, 1, 1)
+	if err != nil {
+		return ret, err
+	}
+	defer rows.Close()
+	for rows.Next() {
+		var orderID string
```

Comment:
Not a big deal here probably, but in general if you keep these variables out of the loop you won't have to allocate new memory each cycle.